src/engine/TextOperationWithoutFilter.cpp

```diff
@@ -12,7 +12,7 @@ using std::string;
 
 // _____________________________________________________________________________
 size_t TextOperationWithoutFilter::getResultWidth() const {
-  size_t width = 2 + getNofVars();
+  size_t width = 3 + getNofVars();
```

Comment:
How difficult would it be, to only add the words, if they are explicitly requested by the query? I am not sure whether this is important, but we anyway have to talk about the semantics of the feature.